services/web/server/tests/unit/with_dbs/02/test_projects_comments_handlers.py

```diff
@@ -145,43 +145,30 @@ async def test_project_comments_full_workflow(
 
     # Now we will log as a different user
     async with LoggedUser(client) as new_logged_user:
-        # As this user does not have access to the project, they should get 404
+        # As this user does not have access to the project, they should get 401
```

Comment:
Q: are you sure about this change? Will the fronted react in the same way?

Reply:
fixed